### PR TITLE
fix: (Platform) fix combobox styles

### PR DESCRIPTION
**Pull request by Betrozov**
#### Please provide a link to the associated issue.
https://github.com/SAP/fundamental-ngx/issues/4251
#### Please provide a brief summary of this pull request.
Fixed 2d part of issue `Combobox is missing styles`
Added `overlay-prebuilt.css` and `popover.css`
#### Please check whether the PR fulfills the following requirements

- [X] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [X] tests for the changes that have been done
- [X] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [X] update `README.md`
- [X] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [X] Documentation Examples
- [X] Stackblitz works for all examples

### CODE REVIEW

**Review by olshansky — APPROVED** [review]

File: libs/platform/src/lib/components/form/combobox/combobox/combobox.component.scss [hunk 1]
```diff
@@ -1,3 +1,6 @@
+@import "~@angular/cdk/overlay-prebuilt";
```
Comment (sKudum) [inline libs/platform/src/lib/components/form/combobox/combobox/combobox.component.scss:1]: Are we suppose to use fiori specifications for styles. Is it temporary import?

**Review by sKudum — APPROVED** [review]

File: libs/platform/src/lib/components/form/combobox/combobox/combobox.component.scss (hunk 1 above)
Comment (sKudum) [reply libs/platform/src/lib/components/form/combobox/combobox/combobox.component.scss:1]: Please ignore, I got it why we have this import. Thank You.approved

Comment (fkolar) [inline libs/platform/src/lib/components/form/combobox/combobox/combobox.component.scss:1]: This shoudl not be part of the component, overlay is used on other places, it should be some global imports

**Review by fkolar — CHANGES_REQUESTED** [review]

**Comment by fkolar** [comment]
@JKMarkowski 
how do we work with this global style overlay for cdk ? Do we import this on some global level?

**Comment by JKMarkowski** [comment]
> @JKMarkowski
> how do we work with this global style overlay for cdk ? Do we import this on some global level?


Hi @fkolar, we used to force users to import `@import "~@angular/cdk/overlay-prebuilt";` inside `angular.json`, but now it's in `popover-body.component.scss`. I see that combobox in platform doesn't use core's popover, so there is no other way, than just adding it into `combobox.component.scss`
Popover component has been improved it be used with diverged control/body [example](https://fundamental-ngx.netlify.app/#/core/popover#trigger). I encourage to use it - it's very flexible and uses cdk's overlay

**Comment by fkolar** [comment]
> > @JKMarkowski
> > how do we work with this global style overlay for cdk ? Do we import this on some global level?
> 
> Hi @fkolar, we used to force users to import `@import "~@angular/cdk/overlay-prebuilt";` inside `angular.json`, but now it's in `popover-body.component.scss`. I see that combobox in platform doesn't use core's popover, so there is no other way, than just adding it into `combobox.component.scss`
> Popover component has been improved. It can be used with diverged control/body [example](https://fundamental-ngx.netlify.app/#/core/popover#trigger). I encourage to use this component - it's very flexible and uses cdk's overlay

@JKMarkowski ComboBox should be really performance driven component and I am not sure how this popover structure woudl be the overhead to this. Compared to using overlay directly. 

@Betrozov Any idea? Can we use pop over directly, isnt to big overhead?

**Comment by Betrozov** [comment]
> > > @JKMarkowski
> > > how do we work with this global style overlay for cdk ? Do we import this on some global level?
> > 
> > 
> > Hi @fkolar, we used to force users to import `@import "~@angular/cdk/overlay-prebuilt";` inside `angular.json`, but now it's in `popover-body.component.scss`. I see that combobox in platform doesn't use core's popover, so there is no other way, than just adding it into `combobox.component.scss`
> > Popover component has been improved. It can be used with diverged control/body [example](https://fundamental-ngx.netlify.app/#/core/popover#trigger). I encourage to use this component - it's very flexible and uses cdk's overlay
> 
> @JKMarkowski ComboBox should be really performance driven component and I am not sure how this popover structure woudl be the overhead to this. Compared to using overlay directly.
> 
> @Betrozov Any idea? Can we use pop over directly, isnt to big overhead?

Hi @fkolar
I need to figure out how long it takes to switch to using the popup.
I'll back when the investigation will be done.

**Review by fkolar — APPROVED** [review]
Lets merge this and create Separate ISSUE where we can discuss how to integrate common component for CDK, its not that straight forward and simply replace this. We can also investigate as part of the ticket if the popover has all the required functionality or we need to extend it. 

Let' not get stuck and find out that we need to enhance one component before merging this PR.

**Review by KevinOkamoto — APPROVED** [review]

**Review by JKMarkowski — APPROVED** [review]

**Comment by Betrozov** [comment]
The issue for discussion here - https://github.com/SAP/fundamental-ngx/issues/4513

**Pull request merged**